rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/start_scan/action.py

```diff
@@ -0,0 +1,66 @@
+import insightconnect_plugin_runtime
+from .schema import StartScanInput, StartScanOutput, Input, Component
+
+# Custom imports below
+from insightconnect_plugin_runtime.exceptions import PluginException
+import ast
+
+
+class StartScan(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="start_scan", description=Component.DESCRIPTION, input=StartScanInput(), output=StartScanOutput()
+        )
+
+    def asset_search(self, hostnames: str, ips: str):
+        parameters = list()
+        parameters.append(("size", 50))
+        resources = self.connection.ivm_cloud_api.call_api_pages("assets", "POST", 3, parameters)
```

Comment:
We'll want to make sure these IPs/Hostnames are included in the search criteria on the asset search. In most environments returning 3 pages won't mean those IPs/hostnames are returned.  We should let the API do that work for us.